employees/views.py

```diff
@@ -646,7 +646,5 @@ def get_context_data(self, **kwargs: Any) -> dict:
             int(self.request.GET["project"])
```

Comment:
I know that this is not a thing which you added here, but I don't get why exactly it is done here in this way. Probably it is checked if project exists or something like this. We have huge technical debt here :(. But it doesn't matter for you here. I won't say you now to fix it, but once you will get more experienced you should always look at strange things like this and see if it can be done better.

Reply:
Ok. Will try to catch these things in the future.